temporalio/api/history/v1/message_pb2_grpc.pyi

```diff
@@ -1,4 +1,25 @@
 """
 @generated by mypy-protobuf.  Do not edit manually!
 isort:skip_file
+The MIT License
+
+Copyright (c) 2020 Temporal Technologies Inc.  All rights reserved.
```

Comment:
Shouldn't this be 2020-2022?

Reply:
This is generated code that was added as the result of a mypy-protobuf update. Specifically https://github.com/nipunn1313/mypy-protobuf/pull/385 now writes our license headers. So this is literally from the top of https://github.com/temporalio/api/blob/master/temporal/api/history/v1/message.proto.

Reply:
We should update there.